#
#  Make sure the first time installed (bootstrap) addons will appear in startup cache.
#  Otherwise they will appear there only after updating the addon and/or application,
#  or after the run with -purgecaches option.
#

--- mozilla/toolkit/mozapps/extensions/internal/XPIInstall.jsm.orig	2022-02-17 14:30:55.153530893 +0300
+++ mozilla/toolkit/mozapps/extensions/internal/XPIInstall.jsm	2022-02-23 01:36:10.047716275 +0300
@@ -1859,8 +1859,9 @@ class AddonInstall {
           extraParams.oldVersion = this.existingAddon.version;
         }
 
         if (this.addon.bootstrap) {
+          flushChromeCaches();
           XPIProvider.callBootstrapMethod(this.addon, file, "install",
                                           reason, extraParams);
         }
 
